vllm/model_executor/models/arctic.py

```diff
@@ -418,13 +417,9 @@ class ArcticForCausalLM(nn.Module, SupportsPP):
     def __init__(self, vllm_config: VllmConfig, prefix: str = "") -> None:
         super().__init__()
         config = vllm_config.model_config.hf_config
-        cache_config = vllm_config.cache_config
         quant_config = vllm_config.quant_config
         self.config = config
-        self.model = ArcticModel(config,
-                                 cache_config,
-                                 quant_config,
-                                 prefix=prefix)
+        self.model = ArcticModel(vllm_config=vllm_config, prefix=prefix)
```

Comment:
I think we should use `maybe_prefix` even in the cases where `prefix` was previously not given, just to be correct. cc @mgoin

Reply:
I don't quite understand this comment. anything else I need to do?

Reply:
Basically, we should also apply `maybe_prefix` for the models where `prefix` argument was newly added.

Reply:
do you mean `maybe_prefix(prefix, "")` ?

Reply:
I mean `maybe_prefix(prefix, <attr_name>)`

Reply:
got your point, fixed in [1ce70da](https://github.com/vllm-project/vllm/pull/10205/commits/1ce70da2b50c0ffda07ad631a453643299297fbf)